src/components/Hero.astro

```diff
@@ -0,0 +1,24 @@
+---
+import Button from './Button.astro';
+import { Image } from 'astro:assets';
+
+const { headline, subhead, buttonsData = [], heroImage, bottomMargin = '' } = Astro.props;
+
+const containerClasses = `container flex flex-wrap mx-auto px-[0.5rem] sm:px-[1rem] md:px-[1.5rem] lg:px-[2rem] xl:px-[1.5rem] 2xl:px-[3.62rem] 3xl:px-[6.62rem] ${bottomMargin}`;
+---
+<div class={containerClasses}>
+  <div class="w-12/12 lg:order-last lg:w-6/12">
```

Comment:
@Mat-Mwi - if you take a look at this at intermediate sizes, the image becomes so large it pushes the text below the fold. Can you please revisit, creating a div to wrap around the text, centered, and shrank per breakpoint so the image and the text is above the fold at small and medium breakpoints?

Reply:
Sorry, I posted https://github.com/fairsource/fair.io/pull/46#pullrequestreview-2266638328 before seeing this. Same feedback. 👍

Reply:
I have addressed this feedback and it is ready for another look. The images width is restricted on smaller breakpoints and it is centered.

Reply:
I'm not sure this fulfills the design intention. We're heading into a 4-day weekend here, let's pick up the conversation next week.

<img width="1133" alt="image" src="https://github.com/user-attachments/assets/1c52acfa-43b6-475f-821a-ebe419a47803">

Reply:
@chadwhitacre - what screen size is this (~992ish - 1100ish)? I feel like it could fit the desktop design.

Reply:
This is at 1023 pixels, one short of the breakpoint.

Reply:
@chadwhitacre Great, we're going to switch to desktop view as Brian suggested and see if that works better. I'm going to see if I can have this ready for review by Tuesday.

Reply:
Sounds good, thanks!